### PR TITLE
Add feature layer extrusion sample

### PR DESCRIPTION
**Pull request by pgruenler**
@dg0yal, can you please review.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Scenes/Feature layer extrusion/FeatureLayerExtrusionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    @IBAction func extrusionAction(_ sender: Any) {
+        // button action for extruding by total population or by population density
+        if showPopulation {
+            renderer?.sceneProperties?.extrusionExpression = "[POP07_SQMI] * 5000"
+            extrusionButton.title = "Total Population"
+        }
+        else {
+            renderer?.sceneProperties?.extrusionExpression = "[POP2007]/ 10"
+            extrusionButton.title = "Population Desity"
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Scenes/Feature layer extrusion/FeatureLayerExtrusionViewController.swift]: Density

Comment (dg0yal) [reply arcgis-ios-sdk-samples/Scenes/Feature layer extrusion/FeatureLayerExtrusionViewController.swift]: please also update screenshot

Comment (pgruenler) [reply arcgis-ios-sdk-samples/Scenes/Feature layer extrusion/FeatureLayerExtrusionViewController.swift]: fixed and updated

**Comment by dg0yal** [comment]
Need to change button to segmented control to make sample more intuitive

**Comment by pgruenler** [comment]
@dg0yal, I made the change we discussed to use a segmented controller.

**Pull request merged**